documentation/my-homeworks/week2/todo-app/src/components/Container.js

```diff
@@ -0,0 +1,29 @@
+import React, { Component } from 'react';
+import Checkbox from './Checkbox'
+import '../App.css';
+
+export default class Container extends Component {
+
+    render() {
+        const {
+            todoItems,
+            handelDoneStatus
+        } = this.props
+
+        return (<div>
+            <ul>
+                {todoItems.map((item, index) => {
+                    return <li key={item.id} className="app-list">
+                        <Checkbox id={item.id}
+                            description={item.description}
```

Comment:
you can pass all the object once using

`<Checkbox item={item} handelDoneStatus={handelDoneStatus} />`